src/interface/hooks/use-post-present-proof-send-request.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * This function is used to send with POST a proposal for a present proof
+ */
+import { useCallback, useState } from 'react'
+import post from '../api/helpers/post'
+
+export default function usePostPresentProofSendRequest() {
+  const path = '/present-proof-2.0/send-request'
+
+  const [error, setError] = useState(null)
+  const [status, setStatus] = useState('idle')
+
+  const createBody = (comment, connectionId, proposal, validity) => {
+    const reqPrs4zkProofs = [
```

Comment:
what does it mean? `Prs4zk`?

Reply:
not sure, will investigate

Reply:
it's just a `pr`edicate`s` `for` `z`ero `k`nowledge. Renamed to `predicates`